include/openssl/conf.h.in

```diff
@@ -31,6 +31,8 @@ use OpenSSL::stackhash qw(generate_stack_macros generate_lhash_macros);
 extern "C" {
 #endif
 
+char *ossl_safe_getenv(const char *);
```

Comment:
I always liked how this uses OPENSSL_issetugid() which is a direct wrapper of issetugid() where available, which has a couple of issues very relevant to the present use that Nico Williams summarized in [this lovely gist](https://gist.github.com/nicowilliams/4daf74a3a0c86848d3cbd9d0cdb5e26e)...

Reply:
Oh, my.  I'm going to add a couple of sentences to doc/ma7/openssl-env

Reply:
Oh, and surely this doesn't belong into a public header.